### PR TITLE
feat: [CDS-45900]: EXPORT_TF_PLAN_JSON_NG ff removal from UI

### PR DESCRIPTION
**Pull request by akhilpandey1**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by akhilpandey1** [comment]
trigger cypress-pipeline

File: src/modules/75-cd/components/PipelineSteps/TerraformPlan/TerraformPlan.tsx [hunk 1]
```diff
@@ -583,7 +581,7 @@ function TerraformPlanWidget(
                           disabled={readonly}
                         />
                       </div>
-                      {EXPORT_TF_PLAN_JSON_NG && (
+                      {
```
Comment (rameshx) [inline src/modules/75-cd/components/PipelineSteps/TerraformPlan/TerraformPlan.tsx:584]: This is fine, but braces can be removed too.

**Review by rameshx — DISMISSED** [review]

File: src/modules/75-cd/components/PipelineSteps/TerraformPlan/__tests__/TerraformPlan.test.tsx [hunk 2]
```diff
@@ -197,13 +197,9 @@ describe('Test TerraformPlan', () => {
     expect(container).toMatchSnapshot()
   })
 
-  test('should render edit view for FF EXPORT_TF_PLAN_JSON_NG enabled', () => {
+  test('should render edit view for FF EXPORT_TF_PLAN_JSON_NG`', () => {
```
Comment (rameshx) [inline src/modules/75-cd/components/PipelineSteps/TerraformPlan/__tests__/TerraformPlan.test.tsx:200]:
```suggestion
  test('should render edit view for FF EXPORT_TF_PLAN_JSON_NG', () => {
```

File: src/modules/75-cd/components/PipelineSteps/TerraformPlan/TerraformPlan.tsx (hunk 1 above)
Comment (akhilpandey1) [reply src/modules/75-cd/components/PipelineSteps/TerraformPlan/TerraformPlan.tsx]: Done

**Review by YounixIO — APPROVED** [review]

**Comment by akhilpandey1** [comment]
retrigger cypress-rest

**Review by YounixIO — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**